salt/metalk8s/addons/ui/files/metalk8s-ui-deployment.yaml

```diff
@@ -27,6 +27,7 @@ spec:
       containers:
         - name: metalk8s-ui
           image: {{ build_image_name('metalk8s-ui', '0.2') }}
+          imagePullPolicy: Always
```

Comment:
unrelated change, shouldn't be in this commit

Reply:
I rename the commit